client/a8c-for-agencies/lib/permission.ts

```diff
@@ -79,6 +81,8 @@ const MEMBER_ACCESSIBLE_PATHS: Record< string, string[] > = {
 	[ A4A_PAYMENT_METHODS_LINK ]: [ 'a4a_jetpack_licensing' ],
 	[ A4A_PAYMENT_METHODS_ADD_LINK ]: [ 'a4a_jetpack_licensing' ],
 	[ A4A_MIGRATIONS_LINK ]: [ 'a4a_read_migrations' ],
+	[ A4A_TEAM_LINK ]: [ 'a4a_read_users' ],
```

Comment:
Not specifically part of the main goal of this PR, but the Team section must also be accessible by team members.